generators/app/templates/src/app/components/Suspense/index.js

```diff
@@ -0,0 +1,26 @@
+import React, { Suspense } from 'react';
+import PropTypes from 'prop-types';
+
+import Loading from '../Spinner/components/loading';
```

Comment:
`components/Loading` ?

Reply:
This component already existed. If we want to change it let's make it in another PR